src/main/webapp/submit-recipe.html

```diff
@@ -57,13 +57,14 @@ <h1 id="title-preview"></h1>
             title: titleInput.value,
             content: instructionsInput.value,
           }),
-        }).then((response) => {
-          if (response.status === 202) {
-            window.location.href = "/feed.html"
-          } else {   
-            alert("Error " + response.status.toString())
-          }
         })
+          .then((response) => {
+            if (response.status !== 200) {
+              alert("Error " + response.status.toString())
```

Comment:
Do we want to return here just so in the case of an error we dont end up having problems with response.json()?

Reply:
Yup, makes sense